packages/edit-site/src/components/sidebar-dataviews/dataview-item.js

```diff
@@ -41,7 +41,7 @@ export default function DataViewItem( {
 	}
 	const linkInfo = useLink( {
 		postType,
-		layout,
+		layout: type,
```

Comment:
What happens if we drop this argument entirely. I think it's not needed right?

Reply:
It is. Otherwise, the dataview & the site editor's layout get out of sync (in the video below, the default trash view & the custom ThemeBuster view use table, but the site editor's layout is not updated accordingly) :

https://github.com/user-attachments/assets/242c4570-fe1a-4b10-81d0-fe1698e2806c